perllib/FixMyStreet/Cobrand/FixMyStreet.pm

```diff
@@ -215,6 +215,16 @@ sub extra_contact_validation {
     return %errors;
 }
 
+sub default_map_zoom {
+    my $self = shift;
+
+    # If we're displaying the map at the user's GPS location we
+    # want to start a bit more zoomed in than if they'd entered
+    # a postcode/address.
+    return $self->{c}->get_param("geolocate") ? 4 : undef;
```

Comment:
As above, wonder if this could be "+1 to default" (so 4 or 3) rather than always 4 (ignoring whether the 2/3 distinction is perfect or not). Just 4 in very rural areas can be a bit blank. But is probably fine?

Reply:
On a separate point, these maps are generated at the command line for emails in the `static_map` function, so $self->{c} will not always be present. Tests don't show this because they involve ImageMagick so that function is skipped.